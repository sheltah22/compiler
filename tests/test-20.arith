let x : <int> = (ref 0) in
let y :  <int> = (ref !x) in
let z :  <[int]> = (ref [] : int) in
z := (!x :: !y :: !z);
!z
