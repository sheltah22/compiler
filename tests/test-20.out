0 :: 0 :: [] : int
